workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -608,8 +611,8 @@ PlayerSettings:
   webGLCompressionFormat: 1
   webGLLinkerTarget: 0
   scriptingDefineSymbols:
-    1: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP
-    4: UNITY_POST_PROCESSING_STACK_V2
+    1: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP;IMPROBABLE_WORKERTYPE_UNITYCLIENT
```

Comment:
Don't know if we need the IMPROBABLE_WORKERTYPE_UNITYCLIENT (see this https://github.com/spatialos/UnityGDK/pull/247#discussion_r212236953 )

Reply:
Good point, I'll recheck that

Reply:
yeah, that can be removed. it's a bug in the build system I believe